src/sub-blocks/Content/ContentList/ContentList.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useContext} from 'react';
+
+import {YFMWrapper} from '../../../components';
+import Image from '../../../components/Image/Image';
+import {getMediaImage} from '../../../components/Media/Image/utils';
+import {ThemeValueContext} from '../../../context/theme/ThemeValueContext';
+import {ContentItemProps, ContentSize} from '../../../models';
+import {block, getThemedValue} from '../../../utils';
+
+import './ContentList.scss';
+
+const b = block('content-list');
+
+interface ContentListProps {
+    contentList: ContentItemProps[];
+    size: ContentSize;
+}
+
+const ContentList = ({contentList, size}: ContentListProps) => {
+    const {themeValue: theme} = useContext(ThemeValueContext);
+
+    return (
+        <div className={b({size})}>
+            {contentList?.map((item) => {
+                const {icon, title, text} = item;
+                const iconThemed = getThemedValue(icon, theme);
+                const iconData = getMediaImage(iconThemed);
+
+                return (
+                    <div className={b('item')} key={title || text}>
```

Comment:
if we dont have title or text we dont have key, and  we'll have error with this example 
`contentList = [{icon: '...'}, {icon: '...'}], {icon: '...'}]]`

Reply:
you can use `import {v4 as uuidv4} from 'uuid';` for key